bridge/client/app/dashboard/dashboard.component.ts

```diff
@@ -12,29 +13,31 @@ import {DataService} from "../_services/data.service";
 })
 export class DashboardComponent implements OnInit, OnDestroy {
 
+  private readonly unsubscribe$ = new Subject<void>();
+
   public projects$: Observable<Project[]>;
```

Comment:
You could already set `projects$` here to `this.dataService.projects`.
by using the rxjs operator `catchError` you could then just return an empty array observable using `of([])` in erroneous cases.
Then, subscribing using the `| async` pipe in the template will always give you an empty array on error.
You won't have to manually subscribe in the component anymore, which means you don't need `unsubscribe$`.
However, just a suggestion, this works perfectly fine too :)

Reply:
generally prefer using `async pipe` above manual subscriptions when possible

Reply:
You are absolutely right, for this use case I can get rid of the manual subscription and provide an empty array as result in erroneous case. Fixed it.
I also log and show errors now in the http-error-interceptor and removed all error handling from the service.